src/BlueprintSourcemap.h

```diff
@@ -101,7 +101,7 @@ namespace snowcrash {
         /** Source Map of Payload-specific Headers */
         SourceMap<Headers> headers;
 
-        /** Source Map of Payload-specific Attributes */
+        /** Source Map of Payload-specific Attributes (THIS SHOULD NOT BE HERE - should be under content) */
```

Comment:
Trust me, This is better for now.

![](https://s3.amazonaws.com/uploads.hipchat.com/25824/153330/U86NXLK5RnbGmce/jakub.png)